dotnet/samples/KernelSyntaxExamples/Example31_CustomPlanner.cs

```diff
@@ -25,8 +25,8 @@ public static async Task RunAsync()
         Console.WriteLine("======== Custom Planner - Create and Execute Markup Plan ========");
         IKernel kernel = InitializeKernel();
 
-        // ContextQuery is part of the QASkill
-        IDictionary<string, ISKFunction> skills = LoadQASkill(kernel);
+        // ContextQuery is part of the QAPlugin
+        IDictionary<string, ISKFunction> skills = LoadQAPlugin(kernel);
```

Comment:
nit: `skills = LoadQAPlugin(kernel)` is a bit confusing if someone is not familiar project.